audit/projects/k8s-infra-ii-sandbox/services/bigquery/bigquery.datasets.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "kind": "bigquery#dataset",
+    "id": "k8s-infra-ii-sandbox:etl_script_generated_set",
+    "datasetReference": {
+      "datasetId": "etl_script_generated_set",
+      "projectId": "k8s-infra-ii-sandbox"
+    },
+    "location": "US"
```

Comment:
@BobyMCbobs Logs activity suggests you made some changes ? 
cc @Riaankl @hh

Reply:
This is from running the ASN data pipeline, to generate the k8s-infra usage costs etc